### ISSUE TITLE
Support for NetCore 2.2 and verionless Microsoft.AspNetCore.App package reference

### ISSUE THREAD

**Issue by latop2604** [szehetner/InliningAnalyzer#13]
When trying to run analysis on asp net core app 2.2, I got lot of FileNotFound errors.  I suspect that's related to new csproj package reference format as version number is not specified for `Microsoft.AspNetCore.App`

Version used : 0.7.4
dotnet --info => 2.2.100

```
Publishing project to C:\Users\me\AppData\Local\Temp\InliningAnalyzer_1
Microsoft (R) Build Engine version 15.9.20+g88f5fadfbe pour .NET Core
Copyright (C) Microsoft Corporation. Tous droits r‚serv‚s.
  MyApi -> C:\Users\me\AppData\Local\Temp\InliningAnalyzer_1\

Starting Inlining Analyzer...
TargetFramework: netcoreapp2.2
Assembly: C:\Users\me\AppData\Local\Temp\InliningAnalyzer_1\MyApi.dll
Runtime: NetCore
Platform: X64
Scope: All Types and Methods

System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.AspNetCore.Mvc.Core, Version=2.2.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'. Le fichier sp‚cifi‚ est introuvable.
File name: 'Microsoft.AspNetCore.Mvc.Core, Version=2.2.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' ---> System.IO.FileNotFoundException: Could not load the specified file.
File name: 'Microsoft.AspNetCore.Mvc.Core'
   at System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
   at System.Runtime.Loader.AssemblyLoadContext.ResolveUsingResolvingEvent(IntPtr gchManagedAssemblyLoadContext, AssemblyName assemblyName)
System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.AspNetCore.Mvc.Core, Version=2.2.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'. Le fichier sp‚cifi‚ est introuvable.
File name: 'Microsoft.AspNetCore.Mvc.Core, Version=2.2.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' ---> System.IO.FileNotFoundException: Could not load the specified file.
File name: 'Microsoft.AspNetCore.Mvc.Core'
   at System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
   at System.Runtime.Loader.AssemblyLoadContext.ResolveUsingResolvingEvent(IntPtr gchManagedAssemblyLoadContext, AssemblyName assemblyName)
...
```

**Comment by szehetner** [comment]
I have just published a new version (0.7.5) where this should be fixed. Please try if this works for you now.

**Comment by latop2604** [comment]
It's working now. Thx. Awesome tool
